src/components/pages/Notes/Note.js

```diff
@@ -9,13 +9,20 @@ import 'simplebar/dist/simplebar.min.css';
 const Note = ({ note }) => {
   const [, , { openDialog }] = useNotes();
   const [expanded, setExpanded] = useState(false);
-  const todoListDiv = content =>
-    JSON.parse(content).map(i => (
+  const todoListDiv = content => {
+    let parsedContent;
+    try {
+      parsedContent = JSON.parse(content);
+    } catch (error) {
+      parsedContent = [];
```

Comment:
It would be nice to have some console.log here, to show that there was an error. What do you think?